### ISSUE TITLE
Case sensitive shenanigans 

### ISSUE THREAD

**Issue by bruno-garcia** [dotnet/nuget-trends#222]
https://nugettrends.com/packages?months=72&ids=SendGrid&ids=Sendgrid

**Comment by meokullu** [comment]
Hi Bruno.
I have to same issue about my package which was named "Prefill" but it is renamed as "PreFill".
https://nugettrends.com/packages?months=12&ids=Prefill

Even though, it is PreFill on NuGet gallery, nuget-trends shows as Prefill with missing data.

**Comment by bruno-garcia** [comment]
> Hi Bruno. I have to same issue about my package which was named "Prefill" but it is renamed as "PreFill". [nugettrends.com/packages?months=12&ids=Prefill](https://nugettrends.com/packages?months=12&ids=Prefill)
> 
> Even though, it is PreFill on NuGet gallery, nuget-trends shows as Prefill with missing data.

Yeah same issue as SendGrid, you can see both for now through adding both ids:

https://nugettrends.com/packages?months=12&ids=Prefill&ids=PreFill

**Comment by bruno-garcia** [comment]
Fixed with the clickhouse migration: https://nugettrends.com/packages?months=72&ids=SendGrid&ids=Sendgrid
